documentation-website/cypress/e2e/autowiring.cy.ts

```diff
@@ -11,7 +11,8 @@ describe('usage/autowiring', () => {
     cy.url().should('eq', BASE_URL + page.autowiring,);
   },);
   it('title matches', () => {
-    cy.title().should('eq', 'Autowiring | Usage | @idrinth/api-bench',);
+    cy.title().should('eq', 'Autowiring | Usage | '
```

Comment:
this package will never exist, please revert the change for now

Reply:
Reverted